src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int EMPTY_INDICATOR = -1;
+    private static final int MAX_SIZE = 12;
+    private int[] NUMBERS = new int[MAX_SIZE];
+    private static final int TOTAL = 11;
+
+    private int total = EMPTY_INDICATOR;
+
+    public void countIn(int in) {
+        if (!isFull())
+            NUMBERS[++total] = in;
+    }
+
+    public boolean isEmpty() {
+        return total == EMPTY_INDICATOR;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).
This constant should be different (separate) than the one in peekaboo and countOut methods.